testData/solutions/web.json

```diff
@@ -64,6 +64,118 @@
+                            "default": {
+                                "outputValue": "Times New Roman"
+                            }
+                        }
+                      }
+                    },
+                    "foregroundColor": {
+                      "transform":{                            
+                        "type": "fluid.transforms.valueMapper",
+                        "inputPath": "http://registry\\.gpii\\.net/applications/com\\.bdigital\\.easit4all/foregroundColor",
+                        "defaultInputValue": "default",
+                        "options": {
+                            "black": {
+                                "outputValue": "Black"
+                            },
+                            "yellow": {
+                                "outputValue": "Yellow"
+                            },
+                            "white": {
+                                "outputValue": "White"
+                            },
+                            "default": {
+                                "outputValue": "Black"
+                            }
+                        }
+                      }
+                    },
+                  "backgroundColor": {
+                      "transform":{                            
+                        "type": "fluid.transforms.valueMapper",
+                        "inputPath": "http://registry\\.gpii\\.net/applications/com\\.bdigital\\.easit4all/backgroundColor",
+                        "defaultInputValue": "default",
+                        "options": {
+                            "black": {
+                                "outputValue": "Black"
+                            },
+                            "yellow": {
+                                "outputValue": "Yellow"
+                            },
+                            "white": {
+                                "outputValue": "White"
+                            },
+                            "default": {
+                                "outputValue": "White"
+                            }
+                        }
+                      }
+                    },
+                  "lineSpacing": "http://registry\\.gpii\\.net/applications/com\\.bdigital\\.easit4all/lineSpacing",
+                  "inputsLarger": "http://registry\\.gpii\\.net/applications/com\\.bdigital\\.easit4all/inputsLarger",
+                  "toc": "http://registry\\.gpii\\.net/applications/com\\.bdigital\\.easit4all/toc",
+                  "links": "http://registry\\.gpii\\.net/applications/com\\.bdigital\\.easit4all/links",                    
+                  "layout": "http://registry\\.gpii\\.net/applications/com\\.bdigital\\.easit4all/layout",
+                  "invertImages": "http://registry\\.gpii\\.net/common/invertColours",
+                  "tracking": "http://registry\\.gpii\\.net/applications/com\\.bdigital\\.easit4all/tracking",
+                  "magnification": {
```

Comment:
This transformation and the associated test fixtures are invalid - since this rule always outputs "magnification": 1 . Is the Easit4All app actually capable of magnification and if so, how is it achieved?